definitions/npm/koa_v2.x.x/flow_v0.94.x-/koa_v2.x.x.js

```diff
@@ -8,7 +8,7 @@
  * breaking: ctx.throw([status], [msg], [properties]) (caused by http-errors (#957) )
 **/
 declare module 'koa' {
-  declare type JSON = | string | number | boolean | null | JSONObject | JSONArray;
+  declare type JSON = | string | number | boolean | null | typeof undefined | JSONObject | JSONArray;
```

Comment:
change to `void` to capture null and undefined

Reply:
Changed `typeof undefined` to `void`, but it seems that that doesn't cover `null`, so I left that in.

Reply:
Huh, unexpected but okay. 😄